src/swapnaWJun20/SumExceptionHandling.java

```diff
@@ -0,0 +1,37 @@
+package swapnaWJun20;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class SumExceptionHandling {
+
+	int getSumThreeNos() {
+		int numbers = 0;
+		int sum = 0;
+		int count = 0;
+	
+		Scanner sc = new Scanner(System.in);
+		System.out.println("How many numbers you want to add : ");
+		int totalNo = sc.nextInt();	
+		while (count < totalNo) {
+			try {
+				System.out.println("Enter the number to add : ");
```

Comment:
Here you should add the number of the integer that user is adding. So something like(count+1) in the sop statement.

Reply:
Suggested changes incorporated in code.